mqtt/mqtt-bridge/src/messages.rs

```diff
@@ -194,10 +230,13 @@ pub async fn retry_subscriptions(
 
     while let Some(subs) = retries.next().await {
         if !subs.is_empty() {
-            info!("try to re-subscribe to {} topics", subs.len());
+            warn!("trying to re-subscribe to {} topics", subs.len());
```

Comment:
This casing looks odd. Do we not use sentence casing anywhere?

Reply:
broker is using lowercase for logging